routemaster/client/assertion_helpers.rb

```diff
@@ -0,0 +1,18 @@
+require 'uri'
+
+module Routemaster
+  class Client
+    module AssertionHelpers
+      def assert_valid_url_throwing_error!(url, error_class)
+        begin
+          uri = URI.parse(url)
+          unless uri.is_a? URI::HTTPS
+            raise error_class, "url '#{url}' is invalid, must be an https url"
```

Comment:
Well... technically it's valid, but not HTTPS.

![](https://cdn.meme.am/cache/instances/folder586/44048586.jpg)